packages/pytest-simcore/src/pytest_simcore/postgres_service2.py

```diff
@@ -0,0 +1,148 @@
+"""
+    sets up a docker-compose
+
+IMPORTANT: incompatible with pytest_simcore.docker_compose and pytest_simcore.postgres_service
```

Comment:
do I understand correctly:
- this is for unit testing where no swarm is deployed? thus, this could be used by the webserver unit tests?
- shall we separate the pytest_simcore unit/integration fixtures? (folder wise?)

Reply:
> * this is for unit testing where no swarm is deployed? 
yes, this is for unit-testing with pytest-docker

>thus, this could be used by the webserver unit tests?
yes, you could but probably tests would need to install some extra requirements.
The problem with pytest-simcore package is that *for the moment* it does not add requirement constraints. Therefore in the meantime it is the user's responsiblity to add all extra requirements in the ``requirements/_test.in``

> * shall we separate the pytest_simcore unit/integration fixtures? (folder wise?)
yes, we need to work a bit into that. I called it ``postgres_service2.py`` intentionally until we find a way to merge it with the rest nicely. I have a couple of proposals in mind:

- add [setup-requires](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies) with special reqs dependeing on the target tests
- separation of unit/integration fixtures
- every interface should have a fixture and fakes of the data